src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/MrdTextConstants.kt

```diff
@@ -12,5 +12,6 @@ class MrdTextConstants {
     const val SCORE_NOT_APPLICABLE = "NOT_APPLICABLE"
     const val TICK_CHARACTER = 0x2713.toChar().toString()
     const val NO_NAME_AVAILABLE = "No name available"
+    const val DEFAULT_DATE_TIME_FOR_NULL_VALUE = "1970-01-01T12:00:00.000"
```

Comment:
Make this 00:00:00.  This is the Unix epoch. It's a special date representing a value of 0 in legacy Unix systems. Every date in that system is expressed as a number of seconds/milliseconds from that 0 point.

Reply:
Done